packages/core/src/components/icon/icon.tsx

```diff
@@ -7,8 +7,8 @@
 import * as classNames from "classnames";
 import * as React from "react";
 
+import { IconName } from "@blueprintjs/icons";
 import { Classes, IIntentProps, IProps } from "../../common";
-import { IconName } from "../../generated/iconName";
 
 export { IconName };
 
```

Comment:
I thought we were gonna move this component to `icons` package too?

Reply:
Discussed yesterday -- component depends on core stuff, will need to move it over _maybe_ when it no longer does depend.